src/Cache/Adapter/None.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Utopia\Cache\Adapter;
+
+use Utopia\Cache\Adapter;
+
+class None implements Adapter
```

Comment:
Does the `None` adapter need to do anything else other than what's coded? How should we test it?

Reply:
We should have a dummy test just to make sure no syntax interface error appear